pages/models/CompensationDevices/STATCOM/index.md

```diff
@@ -0,0 +1,115 @@
+
+It is not suited for low-frequency studies (<10 Hz) event though is still accurate, since EMT models are computationally expensive and require a lot of time because greater time steps lead to convergence errors of the simulation. Phasor models exchange some accuracy for much faster simulation times.
+
+The model is not valid either for harmonic studies, as it only considers the fundamental frequency.
+
+## Model description
+
+The following block diagram shows the main components of the STATCOM model:
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/CompensationDevices/STATCOM/EMT_STATCOM_scheme.svg' | relative_url }}"
+     alt="STATCOM schematic."
+     style="float: center; margin-right: 10px; width: 900px;" />
+</div>
+<div align = 'center'>
+Figure 1: Schematic of the EMT STATCOM model. <a href="#1">[1]</a>
+</div>
+<br>
+
+Notice how it follows the same structure as the Grid Following model, with the addition of the DC Voltage regulation. This control is explained in the following subsection, while the rest can be consulted in the [Grid Following page](../../generations/Sources/VSC/EMTGridFollowingVSC/).
+
+### DC Voltage control
+
+The following control is added to the typical Grid Following control scheme in order to maintain the voltage level at the capacitor [[2]](#2). This ensures that it is not being overcharged or discharged. For a capacitor with a capacitance value $$C$$, the following equation relates the power $$P_{dc}$$ and its energy:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ P_{dc} = V_{DC} I_{DC} = V_{DC} C  \frac{dV_{DC}}{dt} $$
+
+</div>
+
+Applying the Laplace transformation, the equation is transformed to the following:
+
+<div style="background-color:rgba(0, 0, 0, 0.0470588); text-align:center; vertical-align: middle; padding:4px 0;">
+
+$$ P_{dc} = V_{DC} C (s V_{DC}) = sCV_{DC}^2 $$
+
+</div>
+
+The following control structure represents the open-loop of the DC voltage control to obtain an active power reference:
+
+<div style="background-color:rgba(0, 0, 0, 0); text-align:center; vertical-align: middle; padding:4px 0;">
+<img src="{{ '/pages/models/CompensationDevices/STATCOM/DCVoltageControlSTATCOM.svg' | relative_url }}"
+     alt="STATCOM DC Voltage Control open-loop diagram block."
+     style="float: center; margin-right: 10px; width: 600px;" />
+</div>
+<div align = 'center'>
+Figure 2: STATCOM DC Voltage Control open-loop diagram block.
+</div>
+<br>
+
+It is important to note that, in case of an overvoltage, the setpoint obtained will be negative since the capacitor would want to discharge to lower the voltage. Depending on the sign convention used in the AC side, which is typically positive for power injection, a sign change might. It can be added as a gain as in the figure above, but it may not be necessary.
```

Comment:
Depending on the sign convention used in the AC side, which is typically positive for power injection, a sign change might. (This sentence looks incomplete. Could you please have a look?)

Reply:
Solved